### DISCUSSION TITLE
How to create the container..?

### DISCUSSION THREAD

**Discussion by arpit314ad** [Budibase/budibase#1951] [Help]
Hi...

Please help me to setup the date picker in the container object.

![01Scroll](https://user-images.githubusercontent.com/86228170/124908737-46136080-e007-11eb-97a4-91ee61808bc3.JPG)

**Comment by shogunpurple** [comment]
@arpit314ad we are going to need a little bit more information. Please try and include as much detail as possible in discussions, so we can debug more easily :)

As mentioned in https://github.com/Budibase/budibase/discussions/1879#discussioncomment-969497 the best way to achieve this is to use a container inside the form.

- What does your custom CSS look like?
- What does the settings on your container look like? From your component tree, it doesn't look like you have actually wrapped the components in a container, just the field group
